### ISSUE TITLE
RunFrozenPython does not handle arguments

### ISSUE THREAD

**Issue by rbn920** [ZoomerAnalytics/xlwings#588]
Usually when you freeze a python script that can take arguments at the command line the EXE file works the same. 
`python foo.py arg1 agr2`
works the same as
`foo.exe arg1 arg2`
This does not seem to work in xlwings.

**Comment by fzumstein** [comment]
Yes, that's indeed a bit awkward. It has come up before (although not here in the issues).

**Comment by rbn920** [comment]
Any chance this bug will be looked at? I understand it isn't very high priority. I'm just wondering if I need to accept that it is not going to happen.
 Thanks

**Comment by fzumstein** [comment]
RunFrozenPython is indeed not a high prio for me, so I wouldn't expect to get to it within the next couple of weeks, sorry.

**Comment by Kamforka** [comment]
Hey @fzumstein, is this feature is still not implemented yet?

**Comment by fzumstein** [comment]
yes, this is still open...

**Comment by jvaldiviezo9** [comment]
Theoretically what should I do to be able to pass arguments to a frozen py file? Is possible to quickly add that feature? I think this is very important to handle different functions from just one freezed py file. Because with Runpython I can select wich function I can use for example:
Runpython ("import main;main.process_csv()")

But how could I call an specific function with RunFrozenPython?
